packages/eslint-plugin/package.json

```diff
@@ -39,6 +39,7 @@
     "@typescript-eslint/utils": "^5.58.0",
     "chalk": "^4.1.0",
     "confusing-browser-globals": "^1.0.9",
+    "jsonc-eslint-parser": "^2.1.0",
```

Comment:
If we are adding the jsonc parser here, maybe we can / should remove the setup in @nx/plugin generators that add it to the workspace's deps?